Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM heroku/heroku:18
```

Comment:
If we wanted to test against all stacks, this could be templated like here:
https://github.com/heroku/heroku-buildpack-ci-redis/pull/10/files#diff-3254677a7917c6c01f55212f86c57fbfR4

(Though doubt worth it for now)

Reply:
Yeah, this buildpack shouldn't have anything that's specific to heroku-18. So I'm not sure it's worth it.